README.md

```diff
@@ -41,6 +41,21 @@ Chimera is currently composed of three Apache Camel Components, available as sub
 - `camel-chimera-rmlmapper` Camel component used to lifting using the [rmlmapper-cefriel](https://github.com/cefriel/rmlmapper-cefriel) library 
 - `camel-chimera-mapping-template` Camel component able to implement both lifting and lowering steps using the [mapping-template](https://github.com/cefriel/mapping-template) library
 
+### Integration with Camel Karavan
```

Comment:
Move this section as a separate README in the `karavan` folder and add a link here. Rename this section as "Chimera Pipeline Configuration", briefly mention all the Camel DSL (with link to [docs](https://camel.apache.org/manual/dsl.html)) and then Karavan (with link to the subfolder).